roofit/roofitcore/inc/RooExtendPdf.h

```diff
@@ -24,7 +24,9 @@ class RooExtendPdf : public RooAbsPdf {
 
   RooExtendPdf() ;
   RooExtendPdf(const char *name, const char *title, RooAbsPdf& pdf,
-          RooAbsReal& norm, const char* rangeName=nullptr) ;
+	       RooAbsReal& norm, const char* rangeName=nullptr) ;
+  RooExtendPdf(const char *name, const char *title, RooAbsPdf& pdf,
```

Comment:
The third commit for the `RooExtendedPdf` will be superseeded by: https://github.com/root-project/root/pull/12036

Reply:
ok, thanks!